lib/jnpr/junos/device.py

```diff
@@ -940,6 +966,14 @@ def __init__(self, *vargs, **kvargs):
     # -----------------------------------------------------------------------
     # Basic device methods
     # -----------------------------------------------------------------------
+    @property
+    def connected(self):
+        return self._connected
```

Comment:
@vnitinv line 959 in `__init__()` initializes `self._connected` to `False` because it's calling the `connected` setter.